docs/azure_jumpstart_ag/manufacturing/contoso_motors/ai_inferencing/_index.md

```diff
@@ -0,0 +1,68 @@
+
+## Architecture
+
+![AI inference flow](./img/ai_flow.png)
+
+From the image above, the Edge AI inference flow can be divided into two configuration steps and three inference steps. 
+
+1. **Model Downloader:** this is a *Kubernetes Job* that's in charge of downloading the binary files of the AI models and the corresponding configuration files. Depending on the model type, various formats can be used, such as .onnx, .hd5, .bin, and others. In general, the binary files contain the model weights and architecture, while the configuration files specify the model properties, such as input/output tensor shapes, number of layers, and more. 
+
+    In this scenario, the models and configurations needed for the OVMS deployment are hosted within the Arc Jumpstart **jsfiles** storage account. During deployment, the **Model Downloader** job pulls these models and configurations by running the **ovms_config.sh** script, which downloads all necessary model files and stores them in the  **ovms-pvc** persistent volume claim for the OVMS pods to access and serve. All models need to be placed and mounted in a particular directory structure and according to the rules described in [OpenVINO - Model Serving](https://docs.OpenVINO.ai/2022.3/ovms_docs_models_repository.html). 
+    
+    For more information, see [job.yaml](https://github.com/microsoft/jumpstart-agora-apps/blob/manufacturing/contoso_manufacturing/operations/charts/ovms/templates/job.yaml) and [ovms_config.sh](https://raw.githubusercontent.com/microsoft/jumpstart-agora-apps/manufacturing/contoso_manufacturing/deployment/configs/ovms_config.sh).
+
+    | Model | Scenario | Links | 
+    | ----- | -------- | ----- | 
+    | Yolo8 | Object detection | [Yolo8 Models](https://docs.ultralytics.com/modes/#introduction) |
+    | Safety-Yolo | Helmet detection | [safety-yolo8.bin](https://jsfiles.blob.core.windows.net/ai-models/safety-yolo8.bin), [safety-yolo8.xml](https://jsfiles.blob.core.windows.net/ai-models/safety-yolo8.xml) | 
+    | Weld-porosity | Weld porosity detection (no weld, weld, porosity) | [weld-porosity-detection-0001.bin](https://jsfiles.blob.core.windows.net/ai-models/weld-porosity-detection-0001.bin), [weld-porosity-detection-0001.xml](https://jsfiles.blob.core.windows.net/ai-models/weld-porosity-detection-0001.xml) | 
+    | Pose-estimation | Human pose estimation | [human-pose-estimation-0007.bin](https://jsfiles.blob.core.windows.net/ai-models/human-pose-estimation-0007.bin), [human-pose-estimation-0007.xml](https://jsfiles.blob.core.windows.net/ai-models/human-pose-estimation-0007.xml) | 
+    | Electricity-forectasting | Forecast electricity usage (only documented) | [time-series-forecasting-electricity-0001.bin](https://jsfiles.blob.core.windows.net/ai-models/time-series-forecasting-electricity-0001.bin), [time-series-forecasting-electricity-0001.xml](https://jsfiles.blob.core.windows.net/ai-models/time-series-forecasting-electricity-0001.xml) | 
```

Comment:
```suggestion

```

Reply:
remove electricity forecasting, it doesnt neatly well with the rest of the story

Reply:
Ok sure no problem